### PR TITLE
Remove unused function

### PR DESCRIPTION
**Pull request by antkmsft**
Remove this unused funtion. `Do` function does the same thing. This function was not removed by mistake.

### CODE REVIEW

**Review by seankane-msft — APPROVED** [review]

**Comment by jhendrixMSFT** [comment]
Looks like this module is missing its `ci.yml` file.

Also, I think we should shorten the directory name to `appconfig` to align with the module name.

**Review by RickWinter — APPROVED** [review]

**Comment by seankane-msft** [comment]
> Looks like this module is missing its `ci.yml` file.
> 
> Also, I think we should shorten the directory name to `appconfig` to align with the module name.

You can copy the tables ci here: https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/data/aztables/ci.yml

**Comment by antkmsft** [comment]
@jhendrixMSFT, @seankane-msft, thank you, I added ci.yml and renamed `appconfiguration` directory to `appconfig`.

**Comment by seankane-msft** [comment]
> @jhendrixMSFT, @seankane-msft, thank you, I added ci.yml and renamed `appconfiguration` directory to `appconfig`.

Were there also changes to go.mod that need to be made?

**Comment by jhendrixMSFT** [comment]
Oh yes the go.mod will need to be updated so it matches the directory structure.

File: sdk/appconfig/azappconfig/go.mod [hunk 1]
```diff
@@ -0,0 +1,17 @@
+module github.com/Azure/azure-sdk-for-go/sdk/appconfig/azappconfig
```
Comment (jhendrixMSFT) [inline sdk/appconfig/azappconfig/go.mod:1]: go.sum should be included

Comment (antkmsft) [reply sdk/appconfig/azappconfig/go.mod:1]: Thank you, I'll revert.

**Comment by jhendrixMSFT** [comment]
@benbp do we need to run a script to get the pipeline set up?

**Comment by benbp** [comment]
@jhendrixMSFT I just kicked it off ^^

**Comment by benbp** [comment]
@antkmsft FYI the appconfig pipeline just created has some failures: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1414958&view=results

**Pull request merged**